demo/src/main/java/com/novoda/demo/MainActivity.java

```diff
@@ -142,13 +161,19 @@ private void extractFromIntent() {
     @Override
     protected void onStart() {
         super.onStart();
-        Options options = new OptionsBuilder()
-                .withContentType(ContentType.DASH)
-                .withMinDurationBeforeQualityIncreaseInMillis(HALF_A_SECOND_IN_MILLIS)
-                .withMaxInitialBitrate(TWO_MEGABITS)
-                .withMaxVideoBitrate(getMaxVideoBitrate())
-                .build();
-        demoPresenter.startPresenting(mpdAddress, options);
+        offlineLicense.download(new OfflineLicense.OfflineLicenseCallback() {
```

Comment:
NPE in case of `UnsupportedDrmException` when creating the `offlineLicense`. Should we care?

Reply:
I think this would only be the case if the device didn't support widevine which is probably not the case of the people that use this library. Or at least, won't be for much longer considering we will drop `MediaPlayer`.